src/styles/genericPopup.module.scss

```diff
@@ -8,11 +8,11 @@
   z-index: 300;
   .popup {
     text-align: center;
-    margin-inline: auto;
     background-color: $c-gray-dark;
-    position: relative;
+    position: fixed;
     z-index: 200;
     top: 25vh;
+    left: 30vw;
```

Comment:
Better, but still a bit weird placement here. 

How it looks on 1000px width:
![2023-05-20 (9)](https://github.com/saeraa/next-tomten/assets/99484730/63eb865d-f518-4dbe-9ae3-4ffe7bac25e5)